### PR TITLE
Add formatted logging, exported to FOAM context

### PR DESCRIPTION
**Pull request by mdittmer**

### CODE REVIEW

File: lib/org/chromium/webidl/LogLevel.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+  values: [
+    {
+      name: 'DEBUG',
+      shortName: 'DEBG',
+      label: 'Debug',
+      consoleMethodName: 'debug',
+    },
+    {
+      name: 'LOG',
```
Comment (arobins) [inline lib/org/chromium/webidl/LogLevel.js]: This seems weird as a log level.

File: lib/org/chromium/webidl/SimpleLogger.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+      class: 'Function',
+      name: 'getDateString',
+      factory() { return function() { return (new Date()).toString(); } },
+    },
+  ],
+
+  methods: [
+    function put(logLevel) {
+      var args = [logLevel.shortName, '[' + this.getDateString() + ']'].concat(
+        Array.from(arguments).slice(1));
```
Comment (arobins) [inline lib/org/chromium/webidl/SimpleLogger.js]: nit: needs more indent

File: test/any/SimpleLogger-test.js [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+      properties: [
+        ['getDateString', function() { return 'DATE'; }],
+      ],
+    });
+    LogLevel = foam.lookup('org.chromium.webidl.LogLevel');
+    testCtx = foam.lookup('org.chromium.webidl.test.SimpleLogger').create(
+      null,
+      captureLogger =
+        foam.lookup('org.chromium.webidl.test.CaptureLogger').create()
```
Comment (arobins) [inline test/any/SimpleLogger-test.js]: nit: more indent

**Review by arobins — APPROVED** [review]
lgtm, couple minor comments

File: lib/org/chromium/webidl/LogLevel.js (hunk 1 above)
Comment (mdittmer) [reply lib/org/chromium/webidl/LogLevel.js]: Yeah. That's true. I'm following the Javascript `console` interface method names. I suppose I could just drop this one and export `log` as a log-at-info-level. Does that seem more reasonable?

Comment (arobins) [reply lib/org/chromium/webidl/LogLevel.js]: Yes.

**Pull request merged**